app/views/questions/show.html.erb

```diff
@@ -1,8 +1,12 @@
 <h2>Question</h2>
-<% if current_admin? %>
+<% if current_admin? && @question.user.registered_user? %>
```

Comment:
This is nice logic, that the admin can deactivate the user from the question show page.